specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/main.tsp

```diff
@@ -40,5 +40,6 @@ namespace Microsoft.EventGrid.SystemEvents {
   enum ServiceApiVersions {
     v2018_01_01: "2018-01-01",
     v2024_01_01: "2024-01-01",
+    v2025_02_01: "2025-02-01"
```

Comment:
We decided to not add new versions each month. Please revert this and remove `@added` decorator from the various properties.